app/models/vm_or_template/operations/snapshot.rb

```diff
@@ -11,8 +12,9 @@ def validate_create_snapshot
   def validate_remove_snapshot(task = 'Remove')
     return {:available => false, :message => "#{task} Snapshot operation not supported for #{self.class.model_suffix} VM"} unless self.supports_snapshots?
     return {:available => false, :message => "There are no snapshots available for this VM"} if snapshots.size <= 0
-    msg = validate_vm_control
-    return {:available => msg[0], :message => msg[1]} unless msg.nil?
+    unless supports_control?
+      return {:available => false, :message => unsupported_reason(:control)}
```

Comment:
changing this to `return {:available => true, :message => unsupported_reason(:control)}` makes the test pass. But this is only part of the truth.